data/json/recipes/other/materials.json

```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```

Comment:
I recall @Coolthulhu stating that `electronics/parts.json` is the current recommended reference for styling, and it is formatted in this way there. It does not give a reference for when you have a choice of components to use, but it does follow the same conventions for when multiple tools are permitted.